# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/printing/nsPrintData.h
# Commit: f907e3b915ed
# Full Hash: f907e3b915ede8be7e3efc8888ea26fb095a331e
# Author: Jonathan Watt <jwatt@jwatt.org>
# Date: 2022-05-23 09:52:44
# Regressor Bug: 1770539
# File Overlap Count: 1
# Description:
#   Bug 1770539 p6 - Move nsPrintData::mPrintDocList to nsPrintJob. r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D146978
# ==============================================================================

diff -r 4e05361c6b08 -r f907e3b915ed layout/printing/nsPrintData.h
--- a/layout/printing/nsPrintData.h	Sun May 22 09:37:57 2022 +0000
+++ b/layout/printing/nsPrintData.h	Sun May 22 09:37:57 2022 +0000
@@ -14,7 +14,6 @@
 #include "nsDeviceContext.h"
 #include "nsIPrintSettings.h"
 #include "nsISupportsImpl.h"
-#include "nsTArray.h"
 #include "nsCOMArray.h"
 
 class nsPrintObject;
@@ -48,11 +47,6 @@
   // Otherwise, it is unset.
   nsPrintObject* mSelectionRoot = nullptr;
 
-  // Array of non-owning pointers to all the nsPrintObjects owned by this
-  // nsPrintData. This includes this->mPrintObject, as well as all of its
-  // mKids (and their mKids, etc.)
-  nsTArray<nsPrintObject*> mPrintDocList;
-
   bool mOnStartSent;
   bool mIsAborted;  // tells us the document is being aborted
 